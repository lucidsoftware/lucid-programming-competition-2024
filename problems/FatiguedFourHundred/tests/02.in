4
502.77 193.74 165.63 949.12
645.42 820.33 739.32 764.35
396.93 496.82 915.07 202.13
756.67 548.30 856.83 242.34
