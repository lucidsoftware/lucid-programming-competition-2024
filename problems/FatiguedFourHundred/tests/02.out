85 75 75 100 1989.3
80 95 95 100 3285.7
80 90 100 100 2197.0
90 90 100 100 2629.6
